releases/app.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
-  name: vote-app
+  name: js-app
   namespace: arc-k8s-demo
 spec:
   releaseName: arc-k8s-demo
   chart:
-    git: https://github.com/Azure/arc-helm-demo
+    git: https://github.com/akashkeshari/pipelines-javascript-docker
```

Comment:
After this PR gets merged, the charts will be in https://github.com/Azure/arc-helm-demo repository. So this shouldn't be changed

Reply:
yep, cheanged back